### ISSUE TITLE
Sort fails with null values when sorting as number

### ISSUE THREAD

**Issue by chughes-lincoln** [geomoose/gm3#740]
If you have a field that has null values in it, and you sort it as a number in the grid, the results are out of order in the same was as it was in https://github.com/geomoose/gm3/issues/378

Example template in json:
```
{
        "title" : "County Road #",
        "property" : "CountyNum",
        "sortAs" : "number"
 }
```

**Comment by theduckylittle** [comment]
Are you seeing this on 3.10 or on a branch?

**Comment by chughes-lincoln** [comment]
This is on 3.10.

**Comment by klassenjs** [comment]
[template.patch.txt](https://github.com/geomoose/gm3/files/9641834/template.patch.txt)

**Comment by theduckylittle** [comment]
Fixed in commit 81ca6344220edd69cea600b23c17ff96cb19e5d0
